dmci/distributors/pycsw_dist.py

```diff
@@ -67,7 +67,13 @@ def _translate(self):
         try:
             xml_doc = etree.ElementTree(file=self._xml_file)
             transform = etree.XSLT(etree.parse(self._conf.mmd_xslt_path))
-            new_doc = transform(xml_doc)
+            # If the dataset is a parent dataset, the
+            # self._xml_file needs to contain the string "parent"
+            if self._path_to_parent_list:
```

Comment:
Isn't this always true?

Reply:
Well, I guess so. But one of the test did not pass. But I guess I should change the test instead.